lasso/parameters.py

```diff
@@ -621,7 +630,7 @@ def __init__(self, **kwargs):
             self.scratch_location, "make_complete_network_from_fixed_width_file.s"
         )
         self.output_dir = os.path.join(self.scratch_location)
-        self.output_epsg = 26915
+        self.output_epsg = 102646
```

Comment:
@lmz, re: https://app.asana.com/0/12291104512575/1184154758453347/f

The `parameters.py` file in Lasso has all the messy bits in it, including the output projection of the travel model network. We'll edit this file as we go.